homework/Vasili_Zasinets/SetTask/src/test/java/com/tutrit/java/quickstart/homework/set/UniqueWordVocabularyTest.java

```diff
@@ -0,0 +1,37 @@
+package com.tutrit.java.quickstart.homework.set;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class UniqueWordVocabularyTest {
+
+    UniqueWordVocabulary vocabulary;
+    UniqueWordVocabulary vocabularyTest;
+
+    @Before
+    public void setUp() {
+        this.vocabulary = new UniqueWordVocabulary();
+        this.vocabularyTest = new UniqueWordVocabulary();
```

Comment:
My opinion: "this" is redundant

Reply:
I think no

Reply:
Why are you using "this" if you have direct access to the veriables?